var/spack/repos/builtin/packages/lm-sensors/package.py

```diff
@@ -32,3 +32,9 @@ class LmSensors(MakefilePackage):
     def install(self, spec, prefix):
         make('install', 'PREFIX={0}'.format(prefix),
              'ETCDIR={0}/etc'.format(prefix))
+
+    @property
+    def libs(self):
+        return find_libraries(
+            'libsensors', root=self.prefix, shared=True, recursive=True) \
+            or None  # return None to indicate failure
```

Comment:
I don't think this line is needed. By default, `find_libraries` returns an empty `LibraryList()`, which will have the same properties as expected and shouldn't fail when you try to request features from it.

Reply:
I just borrowed this from another package.  I'll remove it.